src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -392,11 +392,19 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             vkey != VK_SNAPSHOT &&
             keyDown)
         {
+            if (modifiers.IsCtrlPressed() && vkey == 'A')

```

Comment:
I think this will impact the way I use my terminal negatively -- _any selection_ will cause `^A` to select all, but I have been known to select text and use `^A` to move to the beginning of the line (emacs key bindings).

Should this be behind a mark mode check?

Reply:
added a mark mode check. I think I had that at one point and got it mixed up. Whoops!

I remember some discussion around it in the selectAll PR, but too lazy to bring that up rn.